doc/changelog.md

```diff
@@ -255,18 +396,29 @@
   shot in a shots-based execution mode.
   [(#731)](https://github.com/PennyLaneAI/catalyst/pull/731)
 
+* The Catalyst frontend now supports Python logging through PennyLane's `qml.logging` module.
+  [(#660)](https://github.com/PennyLaneAI/catalyst/pull/660)
+
+* Catalyst now performs a stricter validation of the wire requirements for devices. In particular,
+  only integer, continuous wire labels starting at 0 are allowed.
+  [(#784)](https://github.com/PennyLaneAI/catalyst/pull/784)
+
+* Catalyst no longer disallows quantum circuits with 0 qubits.
+  [(#784)](https://github.com/PennyLaneAI/catalyst/pull/784)
+
 * Added support for IsingZZ gate in Catalyst frontend. Previously, the IsingZZ gate would be
   decomposed into a CNOT and RZ gates. However, this is not needed as the PennyLane-Lightning
   simulator supports this gate.
   [(#730)](https://github.com/PennyLaneAI/catalyst/pull/730)
 
-* Can now compile functions that have been annotated with return type
-  annotations.
+* Functions that have been annotated with return type
+  annotations will now correctly compile with `@qjit`.
   [(#751)](https://github.com/PennyLaneAI/catalyst/pull/751)
 
-* Refactored `vmap`,`qjit`, `mitigate_with_zne` and gradient decorators in order to follow
+* The `vmap`,`qjit`, `mitigate_with_zne` and gradient decorators now follow
   a unified pattern that uses a callable class implementing the decorator's logic. This
-  prevents having to excessively define functions in a nested fashion.
+  prevents having to excessively define functions in a nested fashion, and allows them
+  to be used as decorators.
   [(#758)](https://github.com/PennyLaneAI/catalyst/pull/758)
   [(#761)](https://github.com/PennyLaneAI/catalyst/pull/761)
   [(#762)](https://github.com/PennyLaneAI/catalyst/pull/762)
```

Comment:
The next entry (PR 712) is internal for sure. Considering it is about tests I would even remove it entirely.

Reply:
Typo for 772:
```
* The built-in instrumentation with `detailed` output will no longer report the cumulative time for
  MLIR pipelines, since the cumulative time was being reported as just another step alongside
  individual timings for each pipeline.
```

Reply:
790 is also an internal change. I would expand it a bit to clarify context:
```
* Measurement process primitives for Catalyst's JAXPR representation now have a standardized
  call signature so that `shots` and `shape` can both be provided as keyword arguments.
```

Reply:
789 is a bit vague:
```
* The finite difference method for differentiation is now always allowed, even on functions with
  mid-circuit measurements, callbacks without custom derivates, or other operations that cannot
  be differentiated via traditional autodiff.
```

Reply:
767 should be highlighted further up I think, being that we officially support a new platform now with this release :)  The part about the github workflow is irrelevant.

Reply:
- I've removed 712 entirely
- 772 fixed
- 790 I've moved to internal changes and updated
- 789 updated
- 767 I've actually updated and moved to the new features section:

  * Catalyst is now officially support on Linux aarch64, with pre-built binaries
  available on PyPI; simply `pip install pennylane-catalyst` on Linux aarch64 systems.
  [(#767)](https://github.com/PennyLaneAI/catalyst/pull/767)